### DISCUSSION TITLE
ofxAssimpModelLoader - adding isLoaded() ?

### DISCUSSION THREAD

**Discussion by dimitre** [openframeworks/openFrameworks#7061] [General]
it would be great to have isLoaded() here too. what do you think?

**Comment by NickHardeman** [comment]
This function is available in the new ofxAssimp add-on :)
https://github.com/openframeworks/openFrameworks/blob/dd09e975ad1166ea0cb48edeae35c1f75dc9ed4b/addons/ofxAssimp/src/ofxAssimpModel.h#L52
